numba_cuda/numba/cuda/decorators.py

```diff
@@ -71,11 +71,12 @@ def jit(func_or_sig=None, device=False, inline=False, link=[], debug=None,
         raise DeprecationError(msg)
 
     debug = config.CUDA_DEBUGINFO_DEFAULT if debug is None else debug
+    opt = (config.OPT._raw_value != 0) if opt is None else opt
```

Comment:
Also needs to avoid using the raw value.

Reply:
removed `_raw_value`.